src/components/feedback/FeedbackForm.tsx

```diff
@@ -28,6 +30,12 @@ export const FeedbackForm = () => {
         },
     });
 
+    const { loggedIn, user } = useAuthContext();
+    const attending = Boolean(user?.Roles?.includes("Attendee"));
+
+    const disabled = !loggedIn || !attending;
```

Comment:
it could be nice to move this down to the JSX itself to avoid the double condition

Reply:
?